docs/database/database-spans.md

```diff
@@ -478,6 +480,28 @@ Semantic conventions for individual database systems or specialized instrumentat
 MAY specify a different `db.query.summary` format as long as produced summary remains
 relatively short and its cardinality remains low comparing to the `db.query.text`.
 
+## Context Propagation
+
+**Status**: [Development][DocumentStatus]
+
+### SQL commenter
+
+Instrumentations SHOULD propagate the context information to the SQL queries following [SQL commenter](https://google.github.io/sqlcommenter/spec/). The instrumentation implementation MAY choose to either **append** the comment to the end of the query or **prepend** the comment at the beginning of the query, depending on the specific database system's requirements or preferences.
+
+| Attribute              | Type   | Description                           | Examples | Require level     | Stability                                                      |
+|------------------------|--------|---------------------------------------|----------|-------------------|----------------------------------------------------------------|
+| [`service.name`](/docs/registry/attributes/service.md) | string | Logical name of the service [22]   | `shoppingcart` | `Recommended`     | ![Development](https://img.shields.io/badge/-development-blue) |
```

Comment:
this table talks about attributes, but it's under context propagation - I don't understand the suggestion here